api/src/Controller/StatsController.php

```diff
@@ -44,4 +45,37 @@ public function getActiveLays()
     {
         return $this->userRepository->findActiveLaysInLastYear();
     }
+
+    /**
+     * @Route("stats/admins/report_data", methods={"GET"})
+     * @Security("is_granted('ROLE_SUPER_ADMIN')")
+     */
+    public function getAdminUserAccountReportData(Request $request, RestFormatter $formatter)
```

Comment:
I think we should add the return type to this function (which would be `array`). As we're writing more modern PHP these days then adding types to function parameters and return types is helpful to everyone when using IDEs :)

Reply:
Just spoke to Alex, although it is an array behind the scenes it will be a response coming back so I have added 'Response' as the return type.

Reply:
My bad, looks like it does return an array and Symfony has some kind of listener that passes the array into a `Response` object. We'll set to `array`.